vectorizing/test/test_vectorizer_heb.py

```diff
@@ -0,0 +1,50 @@
+import logging
+from unittest import TestCase
+import stemming.stemming
+from vectorizing import vectorizer_heb
+from main_flow import main_flow
+from utils import presistor
+
+INSTRUCTIONS_TOP_WORDS = 'inst_top_words.pkl'
+INGREDIENTS_TOP_WORDS = 'ing_top_words.pkl'
+
+
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger('vectorizingEngTest')
+vectorizer = vectorizer_heb.VectorizerHeb()
+stemmer = stemming.stemming.StemmerHebrew()
+main_flow.loadCache()
+top_instruc_dict = presistor.load_parameter_cache_from_disk('app/inst_top_words.pkl')
+top_ingred_dict = presistor.load_parameter_cache_from_disk('app/ing_top_words.pkl')
+
+# please make sure your working directory for each run is RecipeGrabber and not vectorizing/test
+
+
+class TestVectorizerHeb(TestCase):
+
+    def test_example_to_vector_heb1(self):
+        text = "היי תכינו את הבצל ממש ממש טוב ככה יופי טוב שיצא מושלם חם חם מהתנור ואז תדחפו אותו למיקרו ותוסיפו שמן מים וחצילים"
+        result = vectorizer.example_to_vector(text, top_instruc_dict, top_ingred_dict)
+        assert result == []
+
+    def test_example_to_vector_heb2(self):
+        text = "יש לי רק דבר אחד לומר לך הוא מכל הלב"
+        result = vectorizer.example_to_vector(text, top_instruc_dict, top_ingred_dict)
+        assert result == []
+
+    def test_example_to_vector_heb3(self):
+        text = " פתאום תרנגול תרתיחי מים"
+        result = vectorizer.example_to_vector(text, top_instruc_dict, top_ingred_dict)
+        assert result == []
```

Comment:
idk how to write the result here, the vector is 1514 len

Reply:
a fact of life (and a clue) do you know how many bytes in a String of 1 paragraph?

Reply:
found some different way to do it, but i didnt find anything about your idea online. can you teach/send me info about it? thanks!

Reply:
if it work and tests passed so it's ok, 
I just wanted to say it's not different than comparing 2  1514 long strings - no?